spec/03-plugins/16-response-transformer/04-filter_spec.lua

```diff
@@ -97,5 +116,19 @@ for _, strategy in helpers.each_strategy() do
         assert.equals([[this is a / test]], json.uri_args)
       end)
     end)
+
+    describe("unauthorized", function()
+      it("doesn't err when unauthorized", function()
```

Comment:
Within this block, it'd be nice to include a link to the issue this regression test fixes just like https://github.com/Kong/kong/blob/master/spec-old-api/01-unit/010-router_spec.lua#L927

Reply:
We should also be more explicit in the name of this test:
```
it("does not throw an error when request was short-circuited in access phase")
```